package.json

```diff
@@ -4875,6 +4877,622 @@
             }
           }
         ]
+      },
+      {
+        "type": "monovsdbg",
+        "label": ".NET Core using Mono Runtime",
+        "hiddenWhen": "dotnet.debug.serviceBrokerAvailable",
+        "languages": [
+          "csharp"
+        ],
+        "variables": {
+          "pickProcess": "csharp.listProcess",
+          "pickRemoteProcess": "csharp.listRemoteProcess",
+          "pickRemoteDockerProcess": "csharp.listRemoteDockerProcess"
+        },
+        "configurationAttributes": {
+          "launch": {
+            "type": "object",
+            "required": [
+              "program"
+            ],
+            "properties": {
```

Comment:
Do you expect users to ever use `monovsdbg` in a launch.json file? If not, you might not want to document the args.

Reply:
Looking at the implementation, if I understand correctly, a `lanunch` request is really an attach, which means I am assuming a user should never be doing this themselves, so it is probably best to not document the arguments.

Reply:
If users are not going to register anything in a schema, is it good enough to just register the DebugConfigurationProvider and DebugAdapterDescriptorFactory for monosvsdbg?